rffmpeg.py

```diff
@@ -73,6 +73,7 @@
         "logfile": o_config["rffmpeg"]["logging"]["logfile"],
         "remote_hosts": o_config["rffmpeg"]["remote"]["hosts"],
         "remote_user": o_config["rffmpeg"]["remote"]["user"],
+        "remote_persist": o_config["rffmpeg"]["remote"]["persist"],
```

Comment:
For maximum compatibility, this should be done in a conditional like the ones a bit further down, and the default of `0` set if this isn't found in the config. If you're able to implement this I'll wait, if not I'll adjust that quickly after!